packages/notte-browser/src/notte_browser/window.py

```
@@ -106,13 +106,10 @@ def get_chrome_args(self) -> list[str]:
         return chrome_args
 
     @staticmethod
-    def from_request(
-        request: SessionStartRequest,
-        user_agent: str | None = None,
-    ) -> "BrowserWindowOptions":
+    def from_request(request: SessionStartRequest) -> "BrowserWindowOptions":
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)